### ISSUE TITLE
New layouts seem to lose .has-global-padding class

### ISSUE THREAD

**Issue by bgardner** [WordPress/gutenberg#43673]
### Description

On a theme of mine, I am trying to update things to use the new layout controls. When I replace this:

```
<!-- wp:post-content {"layout":{"inherit":true}} /-->
```

With this:

```
<!-- wp:post-content {"layout":{"type":"constrained"}} /-->
```

The result is that the original markup of:
```
<div class="has-global-padding is-layout-constrained entry-content wp-block-post-content">
```

Becomes this:

```
<div class="is-layout-constrained entry-content wp-block-post-content">
```

### Step-by-step reproduction instructions

See above

### Environment info

WordPress 6.0.1, Gutenberg 14.0 RC2

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ndiego** [comment]
@tellthemachines and @andrewserong pinging you both on this. This seems like a bug but figured you both would know more. This issue is related to https://github.com/WordPress/twentytwentythree/pull/121

**Comment by mikachan** [comment]
Confirming that I can also see this happening on my local site: the `is-global-padding` class is removed when `"layout":{"type":"constrained"}` is enabled.

I'm using WordPress 6.0.1 and the latest GB trunk.

**Comment by bgardner** [comment]
If this is indeed a bug (which I believe it is, and a big one), I'd love to see this addresses and included in the 14.0 release. This significantly impacts Site Padding.

**Comment by andrewserong** [comment]
Thanks for the ping! I believe I have a fix for this in https://github.com/WordPress/gutenberg/pull/43689 if folks would like to take a look.

I'm travelling at the moment, so apologies if I don't get back to feedback on the PR immediately! 🙂
